ckanext/data_qld_theme/templates/package/read_base.html

```diff
@@ -0,0 +1,10 @@
+{% ckan_extends %}
```

Comment:
is this only related to comments plugin? is it at all possible for this to be the default template inside the comments plugins as i don't see anything that would relate to our theme (id's class's etc) to pin it. 

Also should it have a new line at the end?